packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -391,22 +392,23 @@ export function Sidebar({
         className={cn(
           'motion-reduce:_transition-none [transition:background-color_1.5s_ease]',
           menu
-            ? '_fixed _inset-0 _z-10 _bg-black/80 dark:_bg-black/60'
+            ? 'max-md:_fixed max-md:_inset-0 max-md:_z-10 max-md:_bg-black/80 max-md:dark:_bg-black/60'
             : '_bg-transparent'
         )}
         onClick={() => setMenu(false)}
       />
       <aside
         className={cn(
           'nextra-sidebar-container _flex _flex-col',
-          'md:_top-16 md:_shrink-0 motion-reduce:_transform-none',
+          'md:_top-16 md:_shrink-0 motion-reduce:_transform-none motion-reduce:_transition-none',
           '_transform-gpu _transition-all _ease-in-out',
           'print:_hidden',
           showSidebar ? 'md:_w-64' : 'md:_w-20',
           asPopover ? 'md:_hidden' : 'md:_sticky md:_self-start',
           menu
             ? 'max-md:[transform:translate3d(0,0,0)]'
-            : 'max-md:[transform:translate3d(0,-100%,0)]'
+            : 'max-md:[transform:translate3d(0,-100%,0)]',
+          isWindowResizing && 'max-md:_transition-none _transition-none'
```

Comment:
we can keep the previous polyfill and add
```suggestion
          '[.resizing_&]:_transition-none'
```

I will push my changes to your branch